docs/content/preview/troubleshoot/cluster/failed_tablets.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Recover YB-TServer from crash loop
+linkTitle: Recover YB-TServer from crash loop
+description: Recover YB-TServer from crash loop
+menu:
+  preview:
+    parent: troubleshoot-cluster
+    weight: 866
+isTocNested: true
+showAsideToc: true
+---
+
+When a YB-TServer process or node has failed, YugabyteDB will automatically trigger a remote bootstrap for
+ most types of tablet data corruption or failures. However, there are still a number of cases where this automatic mechanism might not be able to help.
+
+One major such case is anything that leads to a code path that we've encoded as a crash (eg: `CHECK` or `FATAL`) or is an unknown unknown 
+that causes a crash (eg: code bugs leading to `SIGSEGV`). In all of these cases, it's likely the root cause would repeat itself, when the process is brought back up! 
+Moreover, when using YugabyteDB Anywhere, crashed processes will be automatically restarted, to ensure minimum downtime! 
+
+The problem with servers stuck in such a crash loop though, is that they would likely not be up long enough for us to be able to safely issue runtime commands against them. 
+At that point, manual admin intervention is required, to bring a yb-tserver back to a healthy state!
+
+For this, we'll find all the bad tablets, look for their data on disk (note: this may be spread across multiple disks, depending on your `fs_data_dirs`), and then remove it.
```

Comment:
"For this, we'll find all the bad tablets, look for their data on disk (note: this may be spread across multiple disks, depending on your `fs_data_dirs`), and then remove it.
Here are the steps to address this scenario:"   >

To do this, the administrator needs to find all the faulty tablets, look for their data on disk (possibly spread across multiple disks, depending on your `fs_data_dirs`), and then remove it.
The following are the steps to address this scenario: